tests/console/sshd.pm

```diff
@@ -29,13 +30,8 @@ sub run() {
     assert_screen 'test-sshd-1';
 
     # create a new user to test sshd
-    script_run("useradd -m $ssh_testman");
-    script_run("passwd $ssh_testman; echo password-done > /dev/$serialdev", 0);
-    type_string "$ssh_testman_passwd\n";
-    assert_screen "retry-new-password", 5;
-    type_string "$ssh_testman_passwd\n";
-    wait_serial('password-done') || die "password not set";
-
+    script_run("useradd -m $ssh_testman",     0);
```

Comment:
why do you pass a 0 here? Doesn't make sense to me, we want to wait for the user to be created